assets/dns-app/resources/dns.yaml

```diff
@@ -289,3 +305,685 @@ spec:
+        [ 2192, 138 ],
+        [ 2208, 139 ],
+        [ 2224, 140 ],
+        [ 2240, 141 ],
+        [ 2256, 142 ],
+        [ 2272, 143 ],
+        [ 2288, 144 ],
+        [ 2304, 145 ],
+        [ 2320, 146 ],
+        [ 2336, 147 ],
+        [ 2352, 148 ],
+        [ 2368, 149 ],
+        [ 2384, 150 ],
+        [ 2400, 151 ],
+        [ 2416, 152 ],
+        [ 2432, 153 ],
+        [ 2448, 154 ],
+        [ 2464, 155 ],
+        [ 2480, 156 ],
+        [ 2496, 157 ],
+        [ 2512, 158 ],
+        [ 2528, 159 ],
+        [ 2544, 160 ],
+        [ 2560, 161 ],
+        [ 2576, 162 ],
+        [ 2592, 163 ],
+        [ 2608, 164 ],
+        [ 2624, 165 ],
+        [ 2640, 166 ],
+        [ 2656, 167 ],
+        [ 2672, 168 ],
+        [ 2688, 169 ],
+        [ 2704, 170 ],
+        [ 2720, 171 ],
+        [ 2736, 172 ],
+        [ 2752, 173 ],
+        [ 2768, 174 ],
+        [ 2784, 175 ],
+        [ 2800, 176 ],
+        [ 2816, 177 ],
+        [ 2832, 178 ],
+        [ 2848, 179 ],
+        [ 2864, 180 ],
+        [ 2880, 181 ],
+        [ 2896, 182 ],
+        [ 2912, 183 ],
+        [ 2928, 184 ],
+        [ 2944, 185 ],
+        [ 2960, 186 ],
+        [ 2976, 187 ],
+        [ 2992, 188 ],
+        [ 3008, 189 ],
+        [ 3024, 190 ],
+        [ 3040, 191 ],
+        [ 3056, 192 ],
+        [ 3072, 193 ],
+        [ 3088, 194 ],
+        [ 3104, 195 ],
+        [ 3120, 196 ],
+        [ 3136, 197 ],
+        [ 3152, 198 ],
+        [ 3168, 199 ],
+        [ 3184, 200 ],
+        [ 3200, 201 ],
+        [ 3216, 202 ],
+        [ 3232, 203 ],
+        [ 3248, 204 ],
+        [ 3264, 205 ],
+        [ 3280, 206 ],
+        [ 3296, 207 ],
+        [ 3312, 208 ],
+        [ 3328, 209 ],
+        [ 3344, 210 ],
+        [ 3360, 211 ],
+        [ 3376, 212 ],
+        [ 3392, 213 ],
+        [ 3408, 214 ],
+        [ 3424, 215 ],
+        [ 3440, 216 ],
+        [ 3456, 217 ],
+        [ 3472, 218 ],
+        [ 3488, 219 ],
+        [ 3504, 220 ],
+        [ 3520, 221 ],
+        [ 3536, 222 ],
+        [ 3552, 223 ],
+        [ 3568, 224 ],
+        [ 3584, 225 ],
+        [ 3600, 226 ],
+        [ 3616, 227 ],
+        [ 3632, 228 ],
+        [ 3648, 229 ],
+        [ 3664, 230 ],
+        [ 3680, 231 ],
+        [ 3696, 232 ],
+        [ 3712, 233 ],
+        [ 3728, 234 ],
+        [ 3744, 235 ],
+        [ 3760, 236 ],
+        [ 3776, 237 ],
+        [ 3792, 238 ],
+        [ 3808, 239 ],
+        [ 3824, 240 ],
+        [ 3840, 241 ],
+        [ 3856, 242 ],
+        [ 3872, 243 ],
+        [ 3888, 244 ],
+        [ 3904, 245 ],
+        [ 3920, 246 ],
+        [ 3936, 247 ],
+        [ 3952, 248 ],
+        [ 3968, 249 ],
+        [ 3984, 250 ],
+        [ 4000, 251 ],
+        [ 4016, 252 ],
+        [ 4032, 253 ],
+        [ 4048, 254 ],
+        [ 4064, 255 ],
+        [ 4080, 256 ]
+      ]
+    }
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: autoscaler-coredns-worker
+  namespace: kube-system
+  labels:
```

Comment:
I would also mark it with `gravitational.io/critical-pod` label to make sure gravity status shows if there's a problem with it, e.g. like https://github.com/gravitational/gravity/blob/master/assets/site-app/resources/site.yaml#L117.